superset-frontend/spec/javascripts/explore/components/SelectControl_spec.jsx

```diff
@@ -48,58 +48,27 @@ describe('SelectControl', () => {
     wrapper = shallow(<SelectControl {...defaultProps} />);
   });
 
-  it('uses Select in onPasteSelect when freeForm=false', () => {
```

Comment:
The OnPasteSelect is not used with the new Select component. These tests are included later checking whether `allowNewOptions` is `true` or `false` based on the SelectControl `freeForm` option.